cmd/nodecmd/create.go

```diff
@@ -0,0 +1,678 @@
+}
+
+func setOutput(rootBody *hclwrite.Body) {
+	outputEip := rootBody.AppendNewBlock("output", []string{"instance_eip"})
+	outputEipBody := outputEip.Body()
+	outputEipBody.SetAttributeTraversal("value", hcl.Traversal{
+		hcl.TraverseRoot{
+			Name: "aws_eip",
+		},
+		hcl.TraverseAttr{
+			Name: "myeip",
+		},
+		hcl.TraverseAttr{
+			Name: "public_ip",
+		},
+	})
+
+	outputInstanceID := rootBody.AppendNewBlock("output", []string{"instance_id"})
+	outputInstanceIDBody := outputInstanceID.Body()
+	outputInstanceIDBody.SetAttributeTraversal("value", hcl.Traversal{
+		hcl.TraverseRoot{
+			Name: "aws_instance",
+		},
+		hcl.TraverseAttr{
+			Name: "fuji_node[0]",
+		},
+		hcl.TraverseAttr{
+			Name: "id",
+		},
+	})
+}
+
+func createAnsibleHostInventory(inventoryPath, elasticIP, certFilePath string) error {
+	if err := os.MkdirAll(inventoryPath, os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+	inventoryHostsFile := inventoryPath + "/hosts"
+	inventoryFile, err := os.Create(inventoryHostsFile)
+	if err != nil {
+		return err
+	}
+	alias := "aws-node "
+	elasticIPToUse := elasticIP[1 : len(elasticIP)-2]
+	alias += "ansible_host="
+	alias += elasticIPToUse
+	alias += " ansible_user=ubuntu "
+	alias += fmt.Sprintf("ansible_ssh_private_key_file=%s", certFilePath)
+	alias += " ansible_ssh_common_args='-o StrictHostKeyChecking=no'"
+	_, err = inventoryFile.WriteString(alias + "\n")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func runTerraform() (string, string, error) {
+	var instanceID string
+	var elasticIP string
+	if err := exec.Command("terraform", "init").Run(); err != nil {
+		return "", "", err
+	}
+	var stdBuffer bytes.Buffer
+	cmd := exec.Command("terraform", "apply", "-auto-approve")
+	mw := io.MultiWriter(os.Stdout, &stdBuffer)
+	cmd.Stdout = mw
+	cmd.Stderr = mw
+	if err := cmd.Run(); err != nil {
+		return "", "", err
+	}
+
+	instanceIDOutput, err := exec.Command("terraform", "output", "instance_id").Output()
+	if err != nil {
+		return "", "", err
+	}
+	instanceID = string(instanceIDOutput)
+	eipOutput, err := exec.Command("terraform", "output", "instance_eip").Output()
+	if err != nil {
+		return "", "", err
+	}
+	elasticIP = string(eipOutput)
+	return instanceID, elasticIP, nil
+}
+
+func handleCerts(certName string) error {
+	err := os.Chmod(certName, 0o400)
+	if err != nil {
+		return err
+	}
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	certFilePath := homeDir + "/.ssh/" + certName
+	err = os.Rename(certName, certFilePath)
+	if err != nil {
+		return err
+	}
+	var stdBuffer bytes.Buffer
+	cmd := exec.Command("ssh-add", certFilePath)
+	mw := io.MultiWriter(os.Stdout, &stdBuffer)
+	cmd.Stdout = mw
+	cmd.Stderr = mw
+	return cmd.Run()
+}
+
+func runAnsiblePlaybook(inventoryPath string) error {
+	var stdBuffer bytes.Buffer
+	cmd := exec.Command("ansible-playbook", "main.yml", "-i", inventoryPath, "--ssh-extra-args='-o IdentitiesOnly=yes'")
```

Comment:
main.yml should be generated by the tool, not preexisting

Reply:
we should download playbooks and install them on .avalanche-cli. don't depending on the repo

Reply:
or have constants for each playbook and made cli to create it live, whichever bests. what do you think @dasconnor ?

Reply:
I sugest follow https://github.com/ava-labs/avalanche-cli/pull/865#discussion_r1284906244